lib/Units.pm

```diff
@@ -233,14 +235,19 @@ our %known_units = (
 		m       => 3,
 		aliases => ['cups']
 	},
+	gallon => {
+		factor  => 0.00378541,
+		m       => 3,
+		aliases => [ 'gallons', 'gal' ]
+	},
```

Comment:
I think there are some other related units that could be added here.  Pint and quart go with these volume measurements, and are equally standard with cup and gallon.  There are also teaspoon and tablespoon, that are rather common.  If you really want to go the distance there are also pottle, barrel, shot, gill, hogshead, etc.  I haven't heard of some of those before.  Some fun names too!  See [customary US units](https://en.wikipedia.org/wiki/United_States_customary_units).

Reply:
I'll have a hogsheads of mead with lunch today. 😄

Reply:
For those that like extreme edge cases, the Gal is the acceleration of 1 cm/s^2, but was deprecated by the Int. Ast. Union in 1989.  Probably an idea to include a comment that the gallon is US, not Imperial.

https://en.wikipedia.org/wiki/Gal_(unit)
https://www.iau.org/publications/proceedings_rules/units/

Reply:
@duffee In the help file, I put "US " in front of the title for each of gallon, quart, pint, and cup.